src/content/docs/project/code-of-conduct.md

```diff
@@ -59,7 +59,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at kent+coc@doddsfamily.us. All
+reported by contacting the project team at <kent+coc@doddsfamily.us>. All
```

Comment:
This might be another pr - but i wonder who owns this  email? is this really the one we should be using in this repo?

Reply:
I am going to open an issue about this email so we can still merge this pr!! and deal with the COC separately ✨

Reply:
https://github.com/all-contributors/all-contributors/issues/893 opened issue here!